cmd/kube-mgmt/main.go

```diff
@@ -73,6 +75,8 @@ func main() {
 	rootCmd.Flags().BoolVarP(&params.opaAllowInsecure, "opa-allow-insecure", "", false, "allow insecure https connections to OPA")
 	rootCmd.Flags().StringVarP(&params.podName, "pod-name", "", "", "set pod name (required for admission registration ownership)")
 	rootCmd.Flags().StringVarP(&params.podNamespace, "pod-namespace", "", "", "set pod namespace (required for admission registration ownership)")
+	rootCmd.Flags().StringVarP(&params.policyLabel, "policy-label", "", "", "replace label openpolicyagent.org/policy")
+	rootCmd.Flags().StringVarP(&params.policyValue, "policy-value", "", "", "replace value rego")
```

Comment:
How about we make the default values `"openpolicyagent.org/policy"`  and `"rego"`. Then we just have to plumb the values into the configmap watcher.